src/BetterFolders/index.tsx

```diff
@@ -87,24 +97,52 @@ export default createPlugin({
     styles: css,
     Settings,
     SettingsPanel: () => {
-        const [{ closeOnOpen }, setSettings] = Settings.useState();
+        const [{ closeOnOpen, folderIndicatorPosition }, setSettings] = Settings.useState();
 
         return (
-            <FormSwitch
-                description="Close other folders when opening a new folder"
-                checked={closeOnOpen}
-                onChange={(checked) => {
-                    if (checked) {
-                        // close all folders except one
-                        for (const id of Array.from(ExpandedGuildFolderStore.getExpandedFolders()).slice(1)) {
-                            ClientActions.toggleGuildFolderExpand(id);
+            <>
+                <FormSwitch
+                    description="Close other folders when opening a new folder"
+                    checked={closeOnOpen}
+                    onChange={(checked) => {
+                        if (checked) {
+                            // close all folders except one
+                            for (const id of Array.from(ExpandedGuildFolderStore.getExpandedFolders()).slice(1)) {
+                                ClientActions.toggleGuildFolderExpand(id);
+                            }
                         }
-                    }
-                    setSettings({ closeOnOpen: checked });
-                }}
-            >
-                Close on open
-            </FormSwitch>
+                        setSettings({ closeOnOpen: checked });
+                    }}
+                >
+                    Close on open
+                </FormSwitch>
+                <FormItem className={margins.marginBottom20} title="Folder Indicator Position">
+                    <SingleSelect
+                        value={folderIndicatorPosition}
+                        options={[
+                            {
+                                value: FolderIndicatorPosition.TopLeft,
+                                label: "Default (Top Left)",
+                            },
+                            {
+                                value: FolderIndicatorPosition.TopRight,
+                                label: "Top Right",
+                            },
+                            {
+                                value: FolderIndicatorPosition.BottomLeft,
+                                label: "Bottom Left",
+                            },
+                            {
+                                value: FolderIndicatorPosition.BottomRight,
+                                label: "Bottom Right",
+                            },
+                        ]}
+                        onChange={(value) => setSettings({ folderIndicatorPosition: value })}
+                        renderOptionLabel={({ label }) => <PositionLabel name={label} />}
+                        renderOptionValue={([{ label }]) => <PositionLabel name={label} />}
```

Comment:
No need for custom label rendering when all it does is display the label as text.